docs/core/compatibility/library-change-rules.md

```diff
@@ -333,6 +332,10 @@ Changes in this category modify the public surface area of a type. Most of the c
 
 - ❌ **DISALLOWED: Adding the <xref:System.FlagsAttribute> to an enumeration type**
 
+## API removal
```

Comment:
I think it's overkill to have this section in this article AND a separate article in the TOC called "API removal", so I'd suggest we pick one and delete the other.

Reply:
I agree, I don't think we need to have this section "API Removal".